test/functional/core/fileio_spec.lua

```diff
@@ -219,3 +221,64 @@ describe('tmpdir', function()
     ok(len > 4 and len < 256, '4 < len < 256', tostring(len))
   end)
 end)
+
+describe('skip do you really want to write on force write', function()
```

Comment:
there is no need for a separate describe(), this should live in the describe() starting at line 30